data/filters/templates/fandom-cleanup.yaml

```
@@ -0,0 +1,127 @@
+title: "Fandom: Wiki pages interface cleanups"
+contributors:
+  - JohnyP36
+  - BPower0036
```

```suggestion
contributors:
  - BPower0036
  - JohnyP36
  - lightbulbmeow
```

Let's credit @lightbulbmeow as they came up with the rules. Also, let's keep them in alphabetical order